adagrams/game.py

```diff
@@ -1,11 +1,117 @@
+import random
+
 def draw_letters():
-    pass
+    letter_map = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    letter_pool = ['A','B','C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
+
+    result = []
+    
+    for i in range(10):
+        # letter = random.choice(letter_pool)
+        index = random.randint(0, len(letter_pool) - 1)
```

Comment:
Right now your code isn't actually utilizing the weight distribution of the letters. Currently, all letters have the same chance of being selected (they all can only appear their respective limits). `randint` is only randomly selecting from a pool of integers from `0` to `26`. This means that `Z` is just as likely as likely to be selected as `E`. You would need it to select from a pool of integers from `0` to `98` to have the correct distribution.  How could we change your letter pool to consider this weighted distribution?